pynecone/compiler/compiler.py

```diff
@@ -125,6 +125,22 @@ def _compile_components(components: Set[CustomComponent]) -> str:
     )
 
 
+def _compile_tailwind(
+    config: dict,
```

Comment:
I think the type annotation needs to be `Dict` to support 3.7, we can fix in a follow up